reflow/dtls_wrapper/bf_dwrap.c

```diff
@@ -43,19 +58,28 @@ typedef struct BIO_F_DWRAP_CTX_
 
 BIO_METHOD *BIO_f_dwrap(void) 
 {
-   return(&methods_dwrap);
+   BIO_METHOD *meth = BIO_meth_new(BIO_TYPE_DWRAP, "dtls_wrapper");
```

Comment:
I believe, this memory leaks.

Reply:
I will try to create an unit test and check for memory leaks.